rust-sdk/whirlpool/src/tests/program.rs

```diff
@@ -237,3 +343,13 @@ pub async fn setup_position_bundle(
 
     Ok(position_bundle_address)
 }
+
+pub async fn setup_config_and_fee_tiers(ctx: &RpcContext) -> Result<(), Box<dyn Error>> {
```

Comment:
Is this used anywhere? it is not actually setting up any fee tiers and/or configs